includes/alarmHandler.py

```diff
@@ -40,7 +40,11 @@ def processAlarmHandler(typ, freq, data):
 		logging.debug("starting processAlarm async")
 		try:
 			from threading import Thread
-			Thread(target=processAlarm, args=(typ, freq, data)).start()
+			from copy import deepcopy
```

Comment:
Ich überlege gerade ob ein Copy nicht auch beim sequentiellen Alarm sinnvoll wäre.
Angenommen ein Plugin verändert einfach Daten - das nächste Plugin würde dann mit diesen weiter arbeiten, statt mit den "echten"

Reply:
Ich glaube da liegst du richtig. Eigentlich sollten Plugins die Daten doch aber nicht verändern, oder? Insgesamt wäre das aber sicherlich der bessere Weg. Im Moment kann ich aber noch nicht beurteilen, ob der Copy nicht noch irgendwelche Seiteneffekte hat. Stehe immer noch am Anfang mit Python.

Reply:
SOLLTE nicht - aber was hält ein Plugin in der Theorie davon ab?
Wäre halt schlecht, wenn ein Fehler in einem Plugin zu Seiteneffekten der anderen Plugins führt.

Probleme durch ein Copy sehe ich keine - höchstens in der Geschwindigkeit

Reply:
Ich hab das mal umgebaut, aber mein Bauchgefühl sagt mir, dass es so noch nicht ganz sauber umgesetzt ist. Sollte man statt typ, freq und data in der Funktion processAlarm dann auch dctyp, dcfreq und dcdata verwenden?